packages/common/src/events/_internal/PropertyMapper.ts

```diff
@@ -63,7 +63,7 @@ export class EventPropertyMapper extends PropertyMapper<
     } else {
       obj.attendanceType = HubEventAttendanceType.Online;
     }
-    obj.inPersonCapacity = store.addresses?.[0]?.capacity ?? null;
+    obj.inPersonCapacity = store.inPersonCapacity ?? null;
```

Comment:
Minor, but most of what's in this file deals w/ assigning/deriving values that can't be mapped verbatim from the event record (e.g. mapping the capacity from the 0-index record from onlineMeetings array to the `onlineCapacity` property implemented by the entity POJO, or deriving a `boolean` for the entity POJO's `canChangeStatus` from the event record's `permission` hash). It seems we could remove this line, and simply add the string `"inPersonCapacity"` to the [getPropertyMap fn's commonPropNames array](https://github.com/Esri/hub.js/blob/f/9564-remove-address-model/packages/common/src/events/_internal/getPropertyMap.ts#L10) to achieve the same thing.